### ISSUE TITLE
Test fails for mpi_01_C

### ISSUE THREAD

**Issue by jbigot** [pdidev/pdi#193]
In GitLab by @sdonfack on May 28, 2020, 13:52

``` 
$ ctest --output-on-failure -R mpi_01_C
Test project /build
    Start 508: mpi_01_C
1/1 Test #508: mpi_01_C .........................***Failed    0.58 sec
[PDI][10:20:13] *** trace: Plugins to load: 1
[PDI][10:20:13] *** trace: Creating mpi plugin
[PDI][000000][10:20:13] *** trace: Calling `MPI_COMM_WORLD' share. Callbacks to call: 0
[PDI][000000][10:20:13] *** trace: Calling `MPI_COMM_SELF' share. Callbacks to call: 0
[PDI][000000][10:20:13] *** trace: Calling `MPI_COMM_NULL' share. Callbacks to call: 0
[PDI][000000][10:20:13] *** trace: Calling `MPI_COMM_WORLD_F' share. Callbacks to call: 0
[PDI][000000][10:20:13] *** trace: Calling `MPI_COMM_SELF_F' share. Callbacks to call: 0
[PDI][000000][10:20:13] *** trace: Calling `MPI_COMM_NULL_F' share. Callbacks to call: 0
[PDI][MPI][000000][10:20:13] *** info: (MPI) Plugin loaded successfully
[PDI][000000][10:20:13] *** trace: Loaded 1 metadata
[PDI][000000][10:20:13] *** warning: Data is not defined in specification tree
[PDI][000000][10:20:13] *** info: Initialization successful
[PDI][000000][10:20:13] *** trace: Sharing `my_array' Ref with rights: R = true, W = false
[PDI][000000][10:20:13] *** trace: Calling `my_array' share. Callbacks to call: 1
[PDI][000000][10:20:13] *** trace: Calling `my_array' share. Callbacks to call: 1
Abort(269047045) on node 0 (rank 0 in comm 0): Fatal error in PMPI_Comm_rank: Invalid communicator, error stack:
PMPI_Comm_rank(111): MPI_Comm_rank(comm=0xc4000002, rank=0x7fffffff8e94) failed
PMPI_Comm_rank(84).: Invalid communicator
[PDI][MPI][000000][10:20:13] *** info: Closing plugin
[PDI][000000][10:20:13] *** warning: Remaining 1 reference(s) to `my_array' in PDI after program end

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 0 PID 44826 RUNNING AT jean-zay1
=   EXIT STATUS: 5
===================================================================================


0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.64 sec

The following tests FAILED:
	508 - mpi_01_C (Failed)
Errors while running CTest
```

**Comment by jbigot** [comment]
In GitLab by @ksiero on Jun 2, 2020, 09:49

mentioned in commit 57ca62424001d026af4a3f6b6c987cdde9822a37

**Comment by jbigot** [comment]
closed via merge request !249

**Comment by jbigot** [comment]
In GitLab by @ksiero on Jun 2, 2020, 09:59

closed via commit 57ca62424001d026af4a3f6b6c987cdde9822a37
